File: 2486. Append Characters to String to Make Subsequence.py
class Solution:
    def appendCharacters(self, s: str, t: str) -> int:
        i, j =0, 0
        for i in range(len(s)):
            if j<len(t) and s[i]==t[j]:
                j+=1
            else:
                continue
        return len(t)-j

File: 2486. Append Characters to String to Make Subsequence.c
int appendCharacters(char* s, char* t) {
 int j=0;
 int lenS = strlen(s);
 int lenT = strlen(t);
 for(int i=0;i<lenS;i++){
    if(j<lenT && s[i]==t[j]) j+=1;
 }
 return lenT-j;
}
